packages/components/psammead-bulletin/src/index.stories.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { withKnobs, boolean } from '@storybook/addon-knobs';
+import { storiesOf } from '@storybook/react';
+import { withServicesKnob } from '@bbc/psammead-storybook-helpers';
+import notes from '../README.md';
+import { RadioBulletin, TVBulletin, BulletinImage } from './index';
+
```

Comment:
You could a black container to the stories, so we could see the white bg applied the `BulletinWrapper`.

```
const Container = styled.div`
  background-color: black;
  padding: 1rem;
  height: 100vh;
`;

<Container>
 <Radiobulletin ... />
</Container>

<Container>
 <TVBulletein ... />
</Container>
```

Reply:
Good idea, have added this.